### DISCUSSION TITLE
Add device review for Qubino / Shelly

### DISCUSSION THREAD

**Discussion by 33Fraise33** [zwave-js/zwave-js#6632] [General]
Hello,

It would be nice to add Shelly to the vendor list here: https://github.com/zwave-js/node-zwave-js/blob/master/docs/getting-started/device-review.md

It seems looking at @QubinoHelp their current track history they are actively working with the zwavejs community: https://github.com/zwave-js/node-zwave-js/issues?q=author%3AQubinoHelp+

For their implementation I would look at someone else to review this.

The main reason to include this is the good availability in Europe and reasonable price compared to some competitors with similar lineups. 

@AlCalzone can we look into this?

**Comment by AlCalzone** [comment]
I'm getting some of their devices soon. After that I can provide a more honest review.

**Reply by QubinoHelp** [reply]
We would be delighted to offer you the necessary devices for testing purposes. 

Additionally, we welcome any feedback you may have, as it greatly contributes to the enhancement of our devices. Your insights will be highly valued, as your cooperation will help us improve our products.

I have to mention that we're using the HA with Z-Wave JS as the first controller for our testings in the office as we found it the most complete for the Z-Wave and other devices. Many of us are planning to set it up in our homes as well  ;)

@AlCalzone you have my email feel free to reach out if you need any devices for testing. I'll ensure you or your colleagues receive them promptly.

**Comment by Domopanda** [comment]
Hello there, sorry for the dumb question but I'm really not realizing which is the best way to sorting out my empasse.
I'm novice in using HA for my home and I'm testing lots of different brand/product/protocols to learn and develop the best scheme for my need, however after having used mostly wifi devices, I'm facing off with new shelly (ex qubino) wich work only with z-waves however I have no specific hub for that and trying to avoid to have different hub for each standard....
Btw i'm not really getting if I can think to use HA with Z-Wave JS and thats' it or I still need to have the hardware (hub z-wave) ?
In case I need hub, could you help me in finding the best solution in having just one single hub which is able to to interact with different protocol if exist ?
Really sorry in advance for the dumb questions..and tks in advance for any feedback :)
